webpack/scenes/ContentViews/Details/Filters/ContentViewFilterDetails.js

```diff
@@ -44,6 +45,7 @@ const ContentViewFilterDetails = () => {
         {
           {
             package_group: <CVPackageGroupFilterContent cvId={cvId} filterId={filterId} />,
+            rpm: <CVRpmFilterContent filterId={filterId} inclusion={inclusion} />,
```

Comment:
`filterId` is coming through here as a String, but the PropTypes in `CVRpmFilterContent.js` require it to be a Number.  So, options are (a) use `parseInt(filterId, 10)` to guarantee that it will be a number when passed; or (b) modify the PropTypes to accept a String.